lib/matplotlib/lines.py

```diff
@@ -262,9 +262,9 @@ def __str__(self):
         elif self._x is None:
             return "Line2D()"
         elif len(self._x) > 3:
-            return "Line2D((%g,%g),(%g,%g),...,(%g,%g))" % (
-                self._x[0], self._y[0], self._x[0],
-                self._y[0], self._x[-1], self._y[-1])
+            return "Line2D(({:g},{:g}),({:g},{:g}),...,({:g},{:g}))".format(
+                self._x[0], self._y[0], self._x[1],
+                self._y[1], self._x[-1], self._y[-1])
```

Comment:
Note: I think this was wrong before? We were previously printing the string as `Line2D(x0,y0),(x0,y0),...,(xN,yN))`

Reply:
Can we wrap these as x,y pairs so it's clearer?

Reply:
Yep, good suggestion.